Source/LudoEngine/GameEngine.cs

```diff
@@ -65,15 +85,216 @@ public void PlayGame(GameState game)
         {
             var players = game.GetPlayers();
             game.NextPlayer = players[0];
+
+            // break loop when last person has finished
             while (true)
             {
-                //game.MovePieces(game.NextPlayer
-                //    , game.GetPieces(game.NextPlayer)
-                //    , Dice.Roll());
+                // One round.
+                foreach (var p in players)
+                {
+                    var CurrentPlayerPieces = game.GetPieces(p);
+                    bool rolledSix = false;
+
+                    // This will run if gets to roll again (rolled 6)
+                    do
+                    {
+                        var activePieces = CurrentPlayerPieces.Where(x => x.IsActive == true).ToList();
+                        var inactivePieces = CurrentPlayerPieces.Where(x => x.IsActive == false).ToList();
+
+                        Console.Clear();
+                        //här ska vi skriva ut current game state så man vet var pjäserna står.
+                        Menu.PrintPlayerName(p);
+
+                        // Promts user to roll the dice and prints the roll in the console.
+                        Menu.PromtUserToRollDice();
+                        Console.ReadKey();
+                        int roll = Dice.Roll();
+                        roll = 6;
+                        Menu.PrintDiceRoll(p, roll);
+
+                        if (roll == 1 || roll == 6)
+                        {
+                            UserRolledOneOrSix(game, p, activePieces, inactivePieces, roll);
+                        }
+                        if (roll > 1 || roll < 6)
+                        {
+                            UserRolledTwoToFive(game, p, activePieces, inactivePieces, roll);
+                        }
+
+                        rolledSix = (roll == 6) ? true : false;
+                    } while (rolledSix);
+                }
+
+
+
+            }
+        }
+
+        public void UserRolledOneOrSix(GameState game, Player p, List<Piece> activePieces, List<Piece> inactivePieces, int roll)
```

Comment:
Feels like this method is very big and has many responsibilities. You can change some of the if-statements to boolean methods, this way you shorten the method, transfer responsibilities to the new methods and make the code more readable.